sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -30,7 +30,7 @@ SRC_URI="
 S="${WORKDIR}/VTK-${PV}"
 
 LICENSE="BSD LGPL-2"
-SLOT="0"
+SLOT="0/9.0"
```

Comment:
Also have you considered extracting the 9.0 from ${PV} so it's automated? It's the kind of thing that sometime get forgotten on bumps, and it let us copy ebuilds if no other changes.

Reply:
Oh and when doing generalized changes on all versions, it makes sense to do it in a single commit.

Reply:
Good point on automation. I didn't think about it when doing the changes.

Reply:
A variable for it was already in the files for other purposes, SPV in 8.2 and MY_PV in 9.0. I harmonized it and use the same name in both files for easier maintainance.